site/docs/src/json/identity-providers/login/google-request.json

```diff
@@ -1,8 +1,9 @@
 {
   "applicationId": "10000000-0000-0002-0000-000000000001",
   "data": {
-    "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gUS4gR29vZ2xlZSIsImlhdCI6MTUxNjIzOTAyMn0.qapE7hozQns4dyVk8TOz84zbRebv4zJ5XteDGMfq3QQ"
+    "code": "ZSBxGPP9Ghyxd4QzSQeZv_MvIHuy_vBNKRVygPk_cJM",
+    "redirect_uri": "https://login.piedpiper.com/oauth2/callback"
   },
   "identityProviderId": "82339786-3dff-42a6-aac6-1f1ceecb6c46",
   "ipAddress": "192.168.1.42"
-}
\ No newline at end of file
+}
```

Comment:
We should probably show both ways to call this API, when using the popup, the previous version of this file is correct, and then when using a redirect, the new version is correct. So maybe just add a new file and document both methods. 

Assuming Facebook falls into this category as well, so you can double check if we document both examples for that IdP as well.

Reply:
Facebook does fall into this category, but doesn't yet have any documentation.  I update both Google and Facebook with two examples.  I didn't mention popup or redirect because those are really only in the APP.  The API is only concerned with what the user is passing `code` or `token` not how they came by those.  Do you think we should add some messaging for those?

Reply:
The previous version of this file is still a valid use case when using the pop up. 

So at a minimum we need to document both ways to call it, ideally not removing the old example. 

I think it would be valuable to document in the request properties when it makes sense to call with the token vs code.

Reply:
I moved the `json` around a bit and created an include `_identity-provider-login-request-example.adoc`.  For Google and Facebook I show two examples one using `code` and `redirect_uri` and one using `token`.  For other IdPs I show the IdPs specific `json`.